deploy-manage/deploy/elastic-cloud/differences-from-other-elasticsearch-offerings.md

```diff
@@ -102,7 +102,7 @@ This table compares Elasticsearch capabilities between {{ech}} deployments and S
 
 ### Observability
 
-This table compares Observability capabilities between {{ech}} deployments and Serverless projects:
+This table compares Observability capabilities between {{ech}} deployments and Observability Complete Serverless projects. For more information on Observability Logs Essentials Serverless projects, refer to [Observability subscription tiers](../../../solutions/observability/observability-serverless-feature-tiers.md).
```

Comment:
the children yearn for another column

consider changing the title of the column

Reply:
Changing the column title makes sense. I'm hesitant to add a column for logs essentials here because the comparison here is between ECH and Serverless, not necessarily feature tiers and it might add a little more complexity than we want. I noticed Security doesn't mention specific feature tiers here, but not sure if that's the reason.